src/AvaloniaEdit/Search/SearchPanel.cs

```diff
@@ -190,29 +195,23 @@ private SearchPanel()
         }
 
         /// <summary>
-        /// Creates a SearchPanel and installs it to the TextEditor's TextArea.
+        /// Creates a SearchPanel and installs it to the TextArea.
         /// </summary>
-        /// <remarks>This is a convenience wrapper.</remarks>
         public static SearchPanel Install(TextEditor editor)
         {
-            if (editor == null)
-                throw new ArgumentNullException(nameof(editor));
-            return Install(editor.TextArea);
-        }
+            var textArea = editor.TextArea;
+            if (textArea == null) throw new ArgumentNullException(nameof(textArea));
 
-        /// <summary>
-        /// Creates a SearchPanel and installs it to the TextArea.
-        /// </summary>
-        public static SearchPanel Install(TextArea textArea)
```

Comment:
I would like to keep the ability to install directly into a `TextArea`, since it's a standalone control. Its also maintains API compact with the original project.

Reply:
It is still there but it does not automatically install the search if you use TextArea as standalone anymore. Is this important?